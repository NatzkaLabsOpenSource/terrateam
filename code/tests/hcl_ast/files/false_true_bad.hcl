false true
---
false 1
---
true false
---
true 1.2
---
true "hi"
