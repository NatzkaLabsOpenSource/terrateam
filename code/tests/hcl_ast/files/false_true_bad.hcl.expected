Expected (=) for attribute definition or block labels for block definition.
